app/views/shared/_header.html.erb

```diff
@@ -7,12 +7,12 @@
       <div>
         <%= current_user.surname %>
       </div>
-      <%= link_to "New article", new_article_path %>
-      <%= link_to "Sign out", destroy_session_path(:user) %>
+      <%= link_to (I18n.t "articles.paths.new"), new_article_path %>
```

Comment:
Why there are different sections for `actions` and `paths`?

Reply:
Actions - text for submit buttons
Paths - path to the page